src/commcare_cloud/ansible/roles/redis/tasks/setup_cluster.yml

```diff
@@ -6,7 +6,7 @@
   become: yes
 
 - set_fact: 
-    first_master_ip: "{{ hostvars[groups.redis_cluster_master[0]][internal_network_interface_fact].ipv4.address }}"
+    first_master_ip: "{{ hostvars[groups.redis_cluster_master[0]].internal_ipv4.address }}"
```

Comment:
I had to revert this due to an error `'ansible.parsing.yaml.objects.AnsibleUnicode object' has no attribute 'address'`.